loading.py

```diff
@@ -34,64 +34,36 @@ def get_coordinates(data):
     return coordinates
 
 
-def get_paths(data):
+def get_data_properties(data):
     """
-    Get paths to tiles images.
+    Get significant data from JSON file fields.
 
-    data: a dict created from decoded Tiled 1.2 JSON file
-
-    Return a dictionary with modified tile ID as a key and path to file
-    of the image as a value.
-
-    Create a dictionary where tile ID is modified with the number of the
-    tileset used in Tiled 1.2, so it matches the tile number in the tilelist.
+    Take data - raw JSON input and get tile types, paths to images
+    and tile's custom properties.
```

Comment:
Ještě mě napadla taková drobnost. V dokumentačním řetězci máš pořadí na výstupu napsané jako: `types`, `paths` a `properties`. Ale funkce vrací v pořadí jiném: `type`, `properties` a `paths`. A protože vracíme trojici proměnných, mohlo by dojít při použití funkce podle docstringu k záměně.

Reply:
Ano, další pohrobek, díky.